### PR TITLE
Jed dev

### PR DESCRIPTION
**Pull request by JosephBowkett**
Split out the two versions of the JED

### CODE REVIEW

**Comment by JosephBowkett** [comment]
Should probably wait to accept this until after the JSD PR is closed and I can update the FastCAT fetchDeps to the updated JSD version rather than the dev branch.

File: CMakeLists.txt [hunk 1]
```diff
@@ -54,7 +54,7 @@ find_package(YamlCpp REQUIRED 0.6.3)
 include(FetchContent)
 FetchContent_Declare(jsd
     GIT_REPOSITORY https://github.com/nasa-jpl/jsd.git
-    GIT_TAG v1.5.2
+    GIT_TAG jed-dev
```
Comment (alex-brinkman) [inline CMakeLists.txt:57]: Noting TODO once JSD tag is created

**Review by alex-brinkman — COMMENTED** [review]
The implementation looks good to me. Once JSD is updated we can fix the GIT_TAG and merge this in.

**Comment by alex-brinkman** [comment]
Oh, one final thing - Can you please update the `doc/fastcat_device_config_parameters.md` to contain a specification for the YAML fields of the 2 JED devices?

**Comment by JosephBowkett** [comment]
Done (I think)

**Pull request merged**